bemserver_ui/static/scripts/modules/components/tsChart.js

```diff
@@ -69,11 +69,11 @@ class TimeseriesChart extends HTMLDivElement {
         ],
         series: [
             {
-                type: "line",
+                type: 'line',
                 smooth: true,
             },
         ],
-        useUTC: true,
```

Comment:
No need to modify them, I did it because I wanted to test something with UTC before creating my own files for completeness page and I commited it by mistake sorry.